### ISSUE TITLE
Portable text link crashes desk for multiplayer editors

### ISSUE THREAD

**Issue by alexeygolev** [sanity-io/sanity#4409]
**Describe the bug**

A link annotation in portable text field crashes other editors' desks when the same document is opened and the link URL is edited

**To Reproduce**

1. Open a document with a portable text field that has a link annotation in two tabs/windows
2. Select some text and add a link to the text
3. As soon as you start typing the URL the other instance of this document will crash with "Not a valid block" error

**Expected behavior**

The other editors' desk with the same document should not crash

**Screenshots**

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/1342730/233634433-679665e6-2fb1-45b1-98c7-654f5184420b.png">

**Which versions of Sanity are you using?**

```
@sanity/cli (global)          3.9.0 (up to date)
@sanity/eslint-config-studio  2.0.1 (up to date)
@sanity/vision                3.9.0 (up to date)
sanity                        3.9.0 (up to date)
```

**What operating system are you using?**

macOS 13.3.1

**Which versions of Node.js / npm are you running?**

```
8.1.2
v16.13.2
```

**Comment by skogsmaskin** [comment]
Hey @alexeygolev! Thank you for reporting this. We have fixed the problem. Will let you know when released.

**Comment by skogsmaskin** [comment]
@alexeygolev we released a fix for this in [v3.9.1](https://github.com/sanity-io/sanity/releases/tag/v3.9.1). Closing this.
